transport_and_format.asciidoc

```diff
@@ -312,81 +312,83 @@ include::examples/transport_and_format_get_token_list_example.json[]
+The Hub parses the request and sends a response to the client, which optionally contains its own party-id and country-code in the 'OCPI-from-' header.
+The Hub then sends the pushed data to any client implementing the corresponding applicable module, using its own party-id and country-code in the 'OCPI-from-' header.
+The client receiving a PUSH from a Hub (with the Hubs information in the 'OCPI-from-' header) will respond to
+this PUSH with the Hubs party-id and country-code in the 'OCPI-to-' header.
 
-GET SHALL NOT be used in combination with Broadcast push.
+GET SHALL NOT be used in combination with broadcast pushing.
 If the requesting party wants to GET information of which it does not know the receiving party,
-to <<transport_and_format_message_routing_open_routing_request,Open routing request>> MUST be used. (see below)
+an <<transport_and_format_message_routing_open_routing_request,open routing request>> MUST be used. (see below)
 
-Broadcast push SHALL only be used with information that is meant to be send to all other parties. It is useful for things like:
+Broadcast pushing SHALL only be used with information that is meant to be sent to all other parties. It is useful to share data like
 <<mod_tokens.asciidoc#mod_tokens_tokens_module,Tokens>> and <<mod_locations.asciidoc#mod_locations_locations_module,Locations>>,
-but not for <<mod_cdrs.asciidoc#mod_cdrs_cdrs_module,CDRs>> and <<mod_sessions.asciidoc#mod_sessions_sessions_module,Sessions>>.
+but not so much for <<mod_cdrs.asciidoc#mod_cdrs_cdrs_module,CDRs>> and <<mod_sessions.asciidoc#mod_sessions_sessions_module,Sessions>>
+as these pieces of information are specific to only one party and are possibly even protected by GDPR or other laws.
 
-NOTE: For "Client owned objects" the party-id and country-code in the URL segments will still be the original party-id and country-code from the original client sending the Broadcast push to the Hub.
+NOTE: For "client owned objects", the party-id and country-code in the URL segments will still be the original party-id and country-code from the original client sending the Broadcast push to the Hub.
 
 .Example sequence diagram of a broadcast PUT from a CPO to multiple eMSPs.
 image::images/sd_put_boardcast.svg[OCPI Sequence Diagram Hub PUT]
 
 
 [[transport_and_format_message_routing_open_routing_request]]
 ===== Open routing request
-When a Hub has the intelligence to route messages, based on the content of the request,
-or the requesting party does not know the destination of a request, the 'OCPI-to-' headers can be omitted in the request towards a hub.
-The Hub can then decide to which part a request needs to be routed, or that it needs to be broadcasted.
+When a Hub has the intelligence to route messages based on the content of the request,
+or the requesting party does not know the destination of a request, the 'OCPI-to-' header can be omitted in the request towards the Hub.
```

Comment:
is `header` correct? there are 2 headers: `OCPI-to-party-id` and `OCPI-to-country-code`

Reply:
No, it is wrong and you are absolutely right. I'll change it. I'll also update the table headers in chapter 3.1.7.4 which currently use `TO Header` and `FROM Header` in singular form.